### ISSUE TITLE
2.9.1rc1 rhino build does not contain platforms/rhino.js

### ISSUE THREAD

**Issue by MaxiBul** [jshint/jshint#2767]
2.9.1rc1 rhino build (dist/jshint-rhino.js) does not contain platforms/rhino.js file. In result it doesn't work

Plus for work with Rhino, first line in dist file has to be:
///
var window = this;
///

not
//
var window = {};
//

**Comment by JDougherty** [comment]
Indeed, rhino support has been broken for at least six months (as you noted in issue #2308), and as far back as [two years ago](https://github.com/jshint/jshint/issues/645) it was said:

> Rhino version is deprecated

**Comment by MaxiBul** [comment]
If it can be fixed in 1 line, why not add this fix?
Support of Rhino allow to use jshint in any editor, which support execute external program as source validator, without installing any third party huge platforms like Node (portability). Let it be deprecated, but workable, if it can.
